Makefile

```diff
@@ -15,14 +15,20 @@ stop:	## Stop the environment
 bash:	## Go to the backend container
 	docker-compose exec backend bash
 
-psql:   ## Go to the db and make SQL queries
+psql:	## Go to the db and make SQL queries
 	docker-compose exec db psql -U cfp_v3
 
-populate_database: ## Populate database with fake data
+populate_database:	## Populate database with fake data
 	docker-compose exec backend flask populate-database
 
-rebuild: ## Rebuild docker images
+rebuild:	## Rebuild docker images
```

Comment:
What is the reason for these changes?

Reply:
That was helpful when I was doing changes in Dockerfile and wanted to test the changes quickly.

Reply:
I'm asking about a switch from spaces to tabs. It introduces inconsistency here (as part of commands still uses spaces).

What do you think about that @arturtamborski ?

Reply:
Absolutely, Makefile is all about the tabs. There are some places where you can indent with spaces but mixing these in Makefile (which heavily depends on tabs) can bite someone in the future.

Also, one idea for ensuring that is to create `.editorconfig` file :)
```
root = true

# Unix-style newlines with a newline ending every file
[*]
end_of_line = lf
insert_final_newline = true

# Set default charset
[*.{js,py}]
charset = utf-8

# 4 space indentation
[*.py]
indent_style = space
indent_size = 4

# Tab indentation (no size specified)
[Makefile]
indent_style = tab

[{package.json,.travis.yml}]
indent_style = space
indent_size = 2
```